mabwiser/neighbors.py

```diff
@@ -101,10 +118,9 @@ class _Radius(_Neighbors):
     def __init__(self, rng: _BaseRNG, arms: List[Arm], n_jobs: int, backend: Optional[str],
                  lp: Union[_EpsilonGreedy, _Linear, _Popularity, _Random, _Softmax, _ThompsonSampling, _UCB1],
                  radius: Num, metric: str, no_nhood_prob_of_arm=Optional[List]):
-        super().__init__(rng, arms, n_jobs, backend, lp, metric)
+        super().__init__(rng, arms, n_jobs, backend, lp, metric, no_nhood_prob_of_arm)
```

Comment:
can you comment on this update? I think I saw it's mentioned somewhere but couldn't track

Reply:
The LSH algorithm has the possibility of zero neighbors if the dimensions are too high and there aren't enough tables, so I moved the empty neighborhood functionality to the parent Neighbors class.